src/microbe_stage/Microbe.cs

```diff
@@ -1084,6 +1084,8 @@ public override void _Process(float delta)
             // Play movement sound if one isn't already playing.
             if (!movementAudio.Playing)
                 movementAudio.Play();
+
+            movementAudio.UnitDb = GD.Linear2Db(totalMovement.Length() - 15.0f) * 8.0f - 230.0f;
```

Comment:
What's with UnitDb? Don't players have "VolumeDb" that's used elsewhere to control volume?

Reply:
VolumeDB? Wouldn't that apply it to all audio in a bus?

Reply:
No? That's audio source specific. See the jukebox for example that sets it on individual audio players.

Reply:
```
$ ag VolumeDb
src/general/Jukebox.cs
157:            player.Player.VolumeDb = dbValue;
170:        player.VolumeDb = GD.Linear2Db(linearVolume);

src/engine/Settings.cs
374:        AudioServer.SetBusVolumeDb(master, VolumeMaster);
379:        AudioServer.SetBusVolumeDb(music, VolumeMusic);
384:        AudioServer.SetBusVolumeDb(ambiance, VolumeAmbiance);
389:        AudioServer.SetBusVolumeDb(sfx, VolumeSFX);
394:        AudioServer.SetBusVolumeDb(gui, VolumeGUI);
```
VolumeDb is different from SetBuxVolumeDb

Reply:
seems that AudioStreamPlayer3D does not have that, the non-3d counterpart does tho

Reply:
Maybe UnitDb is the equivalent volume control in it, then?

Reply:
"Base sound level unaffected by dampening, in dB."

Reply:
I suppose that probably is the right property to set, then.